src/controllers/projects.js

```diff
@@ -1,189 +1,71 @@
-const fs = require('fs');
-const projects = require('../data/projects.json');
-const employeesData = require('../data/employees.json');
+import Projects from '../models/Projects';
 
-// getAll projects
-const getAllProjects = (req, res) => {
-  res.status(200).json(projects);
-};
-
-// getActive projects
-const getActiveProjects = (req, res) => {
-  const foundProjects = projects.filter((project) => project.isDeleted === false);
-  res.status(200).json(foundProjects);
-};
+// GetAll
+const getAll = async (req, res) => {
+  try {
+    const projects = await Projects.find();
```

Comment:
filter the data based on the query params:
```suggestion
    const projects = await Projects.find(req.query);
```